SerialPrograms/Source/CommonFramework/VideoPipeline/CameraSelector.h

```diff
@@ -21,7 +21,10 @@ class VideoDisplayWidget;
 
 class CameraSelectorWidget;
 
-
+// Handles the state of video selection: video source and resolution.

```

Comment:
Do you mind pulling all of these comment additions/changes into a separate PR? They can go in now while the actual audio stuff will take longer to review (and will need to wait until I have our own FFT).